src/serialization/sb2.js

```diff
@@ -645,6 +717,21 @@ const parseBlock = function (sb2block, addBroadcastMsg, getVariableId, extension
         shadow: false, // No shadow blocks in an SB2 by default.
         children: [] // Store any generated children, flattened in `flatten`.
     };
+
+    // Attach any comments to this block..
+    const commentsForParsedBlock = (comments && typeof commentIndex === 'number' && !isNaN(commentIndex)) ?
+        comments[commentIndex] : null;
+    if (commentsForParsedBlock) {
+        // TODO currently only attaching the last comment to the block if there are multiple...
```

Comment:
Is this something that can happen through the scratch 2 editor, or is it a hacked project thing?

Reply:
It's something that can happen through the 2.0 editor, you can add multiple comments to the same block, but you can only ever see the top one you created, even though the others exist behind it. If you try to move the top comment around to see the other block comments below it, it gets detached from the block and becomes a workspace comment. =/

Reply:
Ah, so the "use the last one in the list" behavior chooses the one that was visible when imported?